core/datasets.go

```diff
@@ -409,7 +409,7 @@ func (r *DatasetRequests) Save(p *SaveParams, res *repo.DatasetRef) (err error)
 		return fmt.Errorf("peername & name are required to update dataset")
 	}
 
-	// if dsp.DataPath == "" && dsp.DataBytes == nil && dsp.Transform == nil {
+	// if dsp.BodyPath == "" && dsp.BodyBytes == nil && dsp.Transform == nil {
 	// 	return fmt.Errorf("either dataBytes, dataPath, or a transform is required to create a dataset")
```

Comment:
Are we updating all commented out code? This should be "either bodyBytes, bodyPath, ...